modules/oadp-ceph-cephfs-restore.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+ifeval::["{context}" == "cephfs"]
+:cephfs:
+endif::[]
+ifeval::["{context}" == "split"]
+:split:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="oadp-ceph-cephfs-restore_{context}"]
+
+ifdef::cephfs[]
+= Restoring data using OADP 1.2 Data Mover and CephFS storage
+You can use OpenShift API for Data Protection (OADP) 1.2 Data Mover to restore data using CephFS storage if the shallow copy feature of CephFS storage was enabled for the back up procedure. The shallow copy feature is not used in the restore procedure.
+endif::cephfs[]
+
+ifdef::split[]
+= Restoring data using OADP 1.2 Data Mover and split valoumes
+You can use OpenShift API for Data Protection (OADP) 1.2 Data Mover to restore data in an environment that has split volumes, if the shallow copy feature of CephFS storage was enabled for the back up procedure. The shallow copy feature is not used in the restore procedure.
+endif::split[]
+
+.Procedure
+
+. Delete the application namespace by running the following command:
++
+[source,terminal]
+----
+$ oc delete vsb -n <app_namespace> --all
+----
+
+. Delete any `VolumeSnapshotContent` CRs that were created during backup by running the following command:
++
+[source,terminal]
+----
+$ oc delete volumesnapshotcontent --all
+----
+
+. Create a `Restore` CR similar to the following:
```

Comment:
Other examples in this doc set say "as in the following example". We might want to choose one or the other for consistency.

```suggestion
. Create a `Restore` CR as in the following example:
```

Reply:
In my reviews, I have consistently gotten the feedback to use either "similar to the following example" or "as shown in the following example." But I think consistency is probably the most important factor here.